cluster/ci/monitoring/mixins/prometheus/ghproxy_alerts.libsonnet

```diff
@@ -10,7 +10,7 @@
             // "this is going to spam us just about 24/7 with the OWNERS thing"
             // TODO: Undo 404 after https://jira.coreos.com/browse/DPTP-447 is done
```

Comment:
Would like to remove 404 as DPTP-447 is done.

But it seems 404 is still kind of high ~ 6%.

https://prometheus-prow-monitoring.svc.ci.openshift.org/graph?g0.range_input=1d&g0.expr=sum(rate(github_request_duration_count%7Bstatus%3D%22404%22%7D%5B5m%5D))%20%2F%20sum(rate(github_request_duration_count%5B5m%5D))%20*%20100&g0.tab=0

Reply:
A rate of 404 does not indicate any error in the system, so I don't think we care to alert on it.